src/app/components/data-entry-grid/index.html

```diff
@@ -233,6 +238,13 @@ <h2>Passing cell editor parameters</h2>
         You can pass inputs to the SKY UX components used in cell editors via the <a href="https://www.ag-grid.com/javascript-grid-column-properties/">column definition's <sky-code>cellEditorParams</sky-code> property.</a> The component properties are nested under <sky-code>skyComponentProperties</sky-code>. All component properties should be supported, and you can see the defined types by importing <sky-code>SkyCellEditorDatepickerProperties</sky-code> or <sky-code>SkyCellEditorAutocompleteProperties</sky-code>. <sky-code>cellEditorParams</sky-code> can also be a function that returns a param object for dynamic editing params. See the demo for examples of using an object or a function.
       </p>
 
+      <h2>Using other SKY UX components in columns</h2>
```

Comment:
Do we need `Using other` in this header? Or could it just be `SKY UX components in columns`?

Also, is `columns` right? The sentence that follows says `cells`, so should this be `cells` too?

Reply:
I think the "Using other" is important. This section comes after a section that talks about the SKY components we include editors and renderers for in the library. This section is to communicate that even though a SKY component might not have an editor provided by us, you can still use any component you want. Editors/renderers are set column-by-column, so every cell in a column will have the same editor/renderer.

Reply:
Keeping `Using other` makes sense. I was looking at the section in isolation and didn't make the connection.